epochStart/bootstrap/process.go

```diff
@@ -510,7 +510,7 @@ func (e *epochStartBootstrap) syncHeadersFrom(meta *block.MetaBlock) (map[string
 		shardIds = append(shardIds, epochStartData.ShardID)
 	}
 
-	if meta.Epoch > 1 { // no need to request genesis block
+	if meta.Epoch > e.startEpoch+1 { // no need to request genesis block
```

Comment:
wondering how many of these we still have to fix

Reply:
hopefully all of them are fixed now.